src/runtime_src/core/common/query_requests.h

```diff
@@ -81,6 +82,7 @@ enum class key_type
   kds_cu_stat,
   kds_scu_stat,
   ps_kernel,
+  xocl_errors,
```

Comment:
Do we need this key?  Can't we just use the "error" key?

Reply:
Yes it will be become one common key "errors" in future